Model/AutoRoute.php

```diff
@@ -50,7 +50,7 @@ public function setType($type)
         $this->setDefault('type', $type);
     }
 
-    public function setRedirectTarget(AutoRouteInterface $redirectRoute)
+    public function setRedirectTarget($redirectRoute)
```

Comment:
Ok. So, you think that I have to revert my commit https://github.com/mmenozzi/RoutingAutoBundle/commit/cf189436c1d8eeacf6e00da4d2757edd642bd46b?